RoddyCore/src/de/dkfz/roddy/core/ExecutionContext.groovy

```diff
@@ -376,26 +375,30 @@ class ExecutionContext {
         this.executingUser = p
     }
 
+    Optional<String> getAccountingProject() {
```

Comment:
ExecutionContext delivers the accountingProject into the Job. No signature-change of Job constructor needed (and thus no API change).